cmd/detectExecuteScan.go

```diff
@@ -383,15 +366,12 @@ func postScanChecksAndReporting(config detectExecuteScanOptions, influx *detectE
 	if err != nil {
 		return errors.Wrapf(err, "failed to check and report scan results")
 	}
-	policyJsonErr := writeIpPolicyJson(config, utils, paths, sys)
+	policyJsonErr, violationCount := writeIpPolicyJson(config, utils, paths, sys)
 	if policyJsonErr != nil {
 		return errors.Wrapf(policyJsonErr, "failed to write IP policy violations json file")
 	}
-	// create Toolrecord file
-	toolRecordFileName, err := createToolRecordDetect("./", config, sys)
```

Comment:
The toolRecordFile should still be created, right? Was it removed on purpose?

Reply:
Looks like I missed it in while resolving merge conflicts. Sorry for that!